### ISSUE TITLE
Asking for computation resource requirement 

### ISSUE THREAD

**Issue by MathematicalModels** [mogolola/neural_optimizer_search#1]
Dear Mogolola, 

I'm very excited to find your implementation of the "Neural Optimizer Search" work, and have already learnt a lot from reading your code.
The author wrote in the paper that "our controller experiments typically converging in less than a day using 100 CPUs", which is still computational inefficient for me. 
Thus I just wonder how much computation resources does your implementation need to run?
I noticed that your work "does not support distributed computing", thus can you share the time to run this repo on one CPU?

Thank you very much!

**Comment by TsingZ0** [comment]
As the author mentioned in the [PDF](https://github.com/mogolola/neural_optimizer_search/blob/master/Reinforce%20project%20report.pdf), "We train the controller network as well as target network on Amazon EC2, with 1 GPU- Tesla V100, 16GB GPU memory, 8 CPU, 61GB memory. ", "within the time budget we could only train 70 episodes." and "which have the accuracy of 0.5219 on the held-out validation set. This result already outperforms SGD with accuracy 0.4897. "
